README.md

```diff
@@ -1 +1,272 @@
+so that changes (for example `apt install`’ing a new system dependency in a `Dockerfile`)
+can be made in the same pull request that relies on them.
+This reduces the need for coordination across multiple repositories.
+See also the `README` there for more on Servo’s testing workflow.
+
+Some other parts of the configuration are not practical to deploy automatically
+when merging a pull request, and require semi-manual steps. They are kept here.
+
+
+## Requirements
+
+Ensure that you have `python3` and `virtulenv` executables in your `$PATH`.
+The commands below will automatically create a virtual environment
+and install Python dependencies in it.
+
+
+## Obtaining administrative Taskcluster credentials
+
+Most (but not all) commands described below require running in a terminal where this has been done.
+
+```sh
+eval `mach creds`
+```
+
+This command should open a new browser tab.
+If that page says “Sign in to create a client”,
+click the “Sign in” button on the top right then “Sign in with GitHub”.
+Click on the save / floppy disk icon on the bottom right.
+
+This exports environment variables that other commands rely on.
+
+
+## `./mach tc`: Taskcluster [roles], [clients], [worker pools], and [hooks]
+
+[roles]: https://community-tc.services.mozilla.com/docs/manual/design/apis/hawk/roles
+[clients]: https://community-tc.services.mozilla.com/docs/manual/access-control/api#clients
+[hooks]: https://community-tc.services.mozilla.com/docs/reference/core/hooks
+[worker pools]: https://community-tc.services.mozilla.com/docs/reference/core/worker-manager
+
+A Taskcluster deployment has a collection of resources such a
+that define its behavior.
+These can all be managed via the Taskcluster API,
+but managing them by hand is error-prone and difficult to track over time.
+This tool exists to manage those resources in a controlled, observable way.
+It does so by making API calls to determine the current state,
+examining this repository to determine the desired state,
+and then “applying” the necessary changes to get from the former to the latter.
+
+Servo shares the [Community TC](https://community-tc.services.mozilla.com/)
+deployment of Taskcluster with a number of other projects.
+This tool manages the Servo-specific parts of the configuration.
+It is inspired by
+[community-tc-config](https://github.com/mozilla/community-tc-config)
+(which manages the non-project-specific parts)
+and uses [`tc-admin`](https://github.com/taskcluster/tc-admin)
+to examine and update the deployment.
+See that library’s documentation for background on how the process works.
+
+
+### Making changes
+
+You should already have an understanding of the resources you would like to modify.
+See the [Taskcluster Documentation](https://community-tc.services.mozilla.com/docs)
+or consult with the Servo team or the Taskcluster team if you need assistance.
+
+First make sure your branch is up to date with `origin/master` and run:
+
+```
+./mach tc diff
+```
```

Comment:
Oddly, on my fork I get a diff
```
$ ./mach tc diff
--- current
+++ generated
@@ -122,26 +122,16 @@ Role=project:servo:decision-task/trusted:
       - generic-worker:cache:*
       - index:insert-task:project.servo.*
       - project:servo:docker-worker-kvm:capability:privileged
       - queue:create-task:highest:proj-servo/*
       - queue:route:index.project.servo.*
       - queue:route:tc-treeherder-staging.v2._/servo-*
       - queue:route:tc-treeherder.v2._/servo-*

-  Role=repo:github.com/servo/servo:*:
-    roleId: repo:github.com/servo/servo:*
-    description:
-      *DO NOT EDIT* - This resource is configured automatically.
-
-
-    scopes:
-      - queue:route:checks
-      - queue:route:statuses
-
   Role=repo:github.com/servo/servo:branch:*:
     roleId: repo:github.com/servo/servo:branch:*
     description:
       *DO NOT EDIT* - This resource is configured automatically.

       Scopes granted for push events to any branch of servo/servo
     scopes: - assume:project:servo:decision-task/trusted
```

Reply:
This is because of this recent change: https://github.com/mozilla/community-tc-config/pull/129

https://github.com/servo/taskcluster-config/pull/15 should fix it.